### ISSUE TITLE
can't convert complex to float

### ISSUE THREAD

**Issue by chauvinSimon** [rlcode/per#2]
Thank you for this great code!

Just one question regarding the line 16 of `prioritized_memory.py`:
Should not **the absolute value of the TD-error** be taken in `return (error + self.e) ** self.a` ?

This is what is done in the original paper.
In addition, if not using `abs()`, I get the error `can't convert complex to float` due to `(error + self.e)` being negative.

**Comment by being-aerys** [comment]
You are correct in saying that the priority value = absolute td error. However, I think the passed error in return (error + self.e) ** self.a ? is already an absolute value. Or at least that is what u are supposed to do if not already.
